great_expectations/experimental/datasources/dynamic_pandas.py

```diff
@@ -312,12 +349,15 @@ def _generate_pandas_data_asset_models(
                 type_field=(f"Literal['{type_name}']", type_name),
                 fields_dict=fields,
                 extra=pydantic.Extra.forbid,
+                model_docstring=signature_tuple.docstring.partition("\n\nParameters")[0]
```

Comment:
This is safer than split because if the string doesn't contain the target characters the first item in the result tuple is the original string.